handler.go

```diff
@@ -160,16 +162,20 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 type Config struct {
-	Schema   *graphql.Schema
-	Pretty   bool
-	GraphiQL bool
+	Schema                *graphql.Schema
+	Pretty                bool
+	GraphiQL              bool
+	EndpointURL           string
+	SubscriptionsEndpoint string
```

Comment:
Instead of making `EndpointURL` and `SubscriptionsEndpoint` top-level configuration fields, it would probably be cleaner to group them under a `GraphiQLConfig` field that can optionally be passed in together with `GraphiQL: true`. Example:

```go
handler.New(&handler.Config{
    Schema: ...,
    GraphiQL: true,
    GraphiQLConfig: &handler.GraphiQLConfig{
        Endpoint: ...,
        SubscriptionsEndpoint: ...,
    }
})
```

Reply:
I noticed that the Apollo server code for GraphiQL mentioned in #31 has a few more configuration variables that might make sense to add here (like additional HTTP headers to be passed to the endpoint for e.g. authentication).